eng/cake/dotnet.cake

```diff
@@ -0,0 +1,198 @@
+// Contains .NET 6-related Cake targets
+
+var ext = IsRunningOnWindows() ? ".exe" : "";
+var dotnetPath = $"./bin/dotnet/dotnet{ext}";
+
+Task("dotnet")
+    .Description("Provisions .NET 6 into bin/dotnet based on eng/Versions.props")
+    .Does(() =>
+    {
+        var binlog = $"artifacts/dotnet-{configuration}.binlog";
+        var settings = new DotNetCoreBuildSettings
+        {
+            MSBuildSettings = new DotNetCoreMSBuildSettings()
+                .EnableBinaryLogger(binlog)
+                .SetConfiguration(configuration),
+        };
+        DotNetCoreBuild("./src/DotNet/DotNet.csproj");
+    });
+
+Task("dotnet-pack")
+    .Description("Build and create .NET 6 NuGet packages")
+    .IsDependentOn("dotnet")
+    .IsDependentOn("dotnet-buildtasks")
+    .Does(()=>
+    {
+
+        RunMSBuildWithLocalDotNet("Microsoft.Maui-net6.sln", (settings) =>
+        {
+            if (settings is MSBuildSettings msbuildSettings)
+            {
+                msbuildSettings
+                    .WithProperty("SymbolPackageFormat", "snupkg")
```

Comment:
Can you set `SymbolPackageFormat=snupkg` globally in `Directory.Build.props`? why is it passed in?

Reply:
probably? I was only setting it because that's how the initial PS1 script for packaging set it

So whoever wrote that was setting it and I was just following in their foot steps :-)